configure

```diff
@@ -1280,7 +1280,7 @@ config UNIX_OR_WIN32 "$unix_or_win32"
 config UNIXLIB "$unixlib"
 config GRAPHLIB "$graphlib"
 
```

Comment:
The changes here are provisional. Not sure how this should be addressed properly.